docs/cypress-testing-library/intro.md

```diff
@@ -35,40 +39,30 @@ and should be added as follows in `tsconfig.json`:
 }
 ```
 
-You can now use all of `DOM Testing Library`'s `getBy`, `getAllBy`, `queryBy`
-and `queryAllBy` commands.
-[See `DOM Testing Library` API for reference](dom-testing-library/api-queries.md)
+You can find [all Library definitions here](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/testing-library__cypress/index.d.ts).
```

Comment:
Can we keep a link to the queries with an explanation of why `get` isn't exposed? TS defs are community maintained and might not be 100% up to date all the time (plus you need to understand Typescript).